src/odemis/gui/comp/stream.py

```diff
@@ -2154,11 +2154,64 @@ def __init__(self, parent,
         """
         wx.Panel.__init__(self, parent, wid, pos, size, style, name)
         self.buttons = {}  # int --> wx.Button
+        # Dwell time slider variables
+        self._dwell_time_slider_conf = None
+        self._dwell_time_slider_ctrl = None
+        self._dwell_time_grid_sz = wx.GridBagSizer()
+        self._dwell_time_grid_sz.SetEmptyCellSize((0, 0))
 
         self._panel_sizer = wx.BoxSizer(wx.VERTICAL)
         self.SetSizer(self._panel_sizer)
 
-    def create_controls(self, layout):
+    @property
+    def dwell_time_slider_conf(self):
+        return self._dwell_time_slider_conf
+
+    @dwell_time_slider_conf.setter
+    def dwell_time_slider_conf(self, conf: dict):
+        self._dwell_time_slider_conf = conf
+
+    @property
+    def dwell_time_slider_ctrl(self):
+        return self._dwell_time_slider_ctrl
+
+    @dwell_time_slider_ctrl.setter
+    def dwell_time_slider_ctrl(self, ctrl):
+        self._dwell_time_slider_ctrl = ctrl
+
+    def _add_dwell_time_slider(self, label_text: str = "Dwell time", value: float = None, conf: dict = None):
+        """ Add an float value slider to the overview acquistion panel
+
+        :param label_text: (str) Label text to display
+        :param value: (None or float) Value to display
```

Comment:
what value is this? Is this the value "selected" by the slider?

Reply:
I think I understand, the slider will be set to this value, right?

Reply:
Right, the slider be set to this init value